docs/pages/api.rst

```diff
@@ -711,20 +711,22 @@ Signature: ``Collection::falsy();``
 filter
 ~~~~~~
 
-Filter collection items based on one or more callbacks.
+Filter collection items based on one or more callbacks. Multiple callbacks will be treated as a logical ``AND``.
```

Comment:
I feel like it's a bit inconsistent with the others that `filter` works like this. I was expecting multiple callbacks to behave like logical `OR`

Reply:
@drupol what do you think?

Reply:
Yep, this is one of the gotcha(s) I couldn't fix in a lazy way.

Reply:
I see, no worries I've updated the docs and example 😁